src/apiCalls.js

```diff
@@ -0,0 +1,9 @@
+export const getResults = (coords) => {
+  return fetch('https://fe-cors-proxy.herokuapp.com', {
```

Comment:
We'll have to request that back-end enables CORS moving forward but, for now, this is a good solution.

Reply:
@TravisGM92 @judithpillado @robertorodriguez12 

When we made our GET request to the API, we were denied access via CORS.  Is there a way to get this set up on the back-end so that we can fetch from your endpoint directly, rather than through our current proxy above?

Reply:
@nicolegooden are you not able to just pull in the URL as such `https://boneyard-be.herokuapp.com/api/coordinates=` and put the coordinates at the end?

Reply:
@TravisGM92 we are able to do that on Postman, but in the browser working on `localhost`, we get something like this error in the console: 

`Access to fetch at '<endpoint>' from origin '<insert another url' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

Reply:
That sounds like a framework issue coming from your end

Reply:
It could be.  @npdarrington mentioned speaking to back-end about this, so maybe he has more context?

Reply:
We are not able to because of CORS error when making the request to the endpoint from an external site (our site) rather than directly on your endpoint's URL. We just need the endpoint to be CORS-enabled. For now we are using a work-around proxy.

Reply:
@TravisGM92 The error we recieve is `No 'Access-Control-Allow-Origin' header is present`. Looks like [this addresses how to solve it](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMissingAllowOrigin)

Reply:
So it's likely not a framework issue coming from your end. I'm doing some research, looks like we'll need to do some things on the BE, but I'll need to try some things out on a FE branch that can get the error to show. So if someone gets on today (1/10), let me know.

Reply:
Bret says you were able to solve the problem. I'll be interested in learning how that gets handled on the server side!